Packs/Jira/ReleaseNotes/1_2_12.md

```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Atlassian Jira v2
+- Fixed an issue where *issue type* was printed under *project name* place in ***jira-create-issue*** war room response.
```

Comment:
```suggestion
- Fixed an issue where the *issue type* was printed instead of the *project name* in the war room response of the command ***jira-create-issue***.
```

Reply:
```suggestion
 - Fixed an issue where the *issue type* was printed instead of the *project name* in the war room response of the ***jira-create-issue*** command.
```